src/data/services/EnterpriseDataApiService.js

```diff
@@ -123,6 +123,48 @@ class EnterpriseDataApiService {
     });
   }
 
+  static fetchEnrolledLearnersForInactiveCourses(options) {
+    const { enterpriseId } = store.getState().portalConfiguration;
+    const queryParams = {
+      page: 1,
+      page_size: 50,
+      has_enrollments: true,
+      active_courses: false,
+      all_enrollments_passed: false,
+      extra_fields: 'enrollment_count+course_completion_count',
+      ...options,
+    };
+
+    const analyticsUrl = `${EnterpriseDataApiService.enterpriseBaseUrl}${enterpriseId}/users/?${qs.stringify(queryParams)}`;
+    const jwtToken = getAccessToken();
+
+    return httpClient.get(analyticsUrl, {
+      headers: {
+        Authorization: `JWT ${jwtToken}`,
+      },
+    });
+  }
+
+  // TODO wire this up with generic DownloadCsvButton component
```

Comment:
We can remove this TODO since we are wiring this up with it's own download csv button.